pyproject.toml

```
@@ -2,7 +2,7 @@
 # Copyright (c) Contributors to the OpenEXR Project.
 
 [build-system]
-requires = ["scikit-build-core==0.8.1"]
+requires = ["scikit-build-core==0.8.1", "pybind11", "numpy"]
```

```suggestion
requires = ["scikit-build-core==0.8.1", "pybind11"]
```
`numpy` is only needed in the build deps if the Numpy's C API is used. From a quick look at the PR, there seems to be no usage of the C API.